app/index.html

```diff
@@ -32,7 +32,6 @@
     <!-- build:css(.tmp) styles/main.css -->
     <link rel="stylesheet" type="text/css" href="styles/main.css">
     <!-- endbuild -->
-    <link rel="stylesheet" type="text/css" href="styles/extensions.css">
```

Comment:
Why not do the same build exclusion for extensions.css so we can still do dev testing of extensions?